src/btree/bt_sync.c

```diff
@@ -271,6 +271,10 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
             if (WT_PAGE_IS_INTERNAL(page)) {
                 internal_bytes += page->memory_footprint;
                 ++internal_pages;
+                /* slow down checkpoints */
```

Comment:
This should be capitalized and have a period at the end: "Slow down checkpoints."